spec/requests/api/v1.0/filter_spec.rb

```diff
@@ -101,6 +101,28 @@ def expect_success_ordered_objects(collection, query, results)
     it("key:eq_i array")  { expect_success("sources", "filter[undocumented][eq_i][]=ABC&filter[undocumented][eq_i][]=XYZ", source_1, source_2, source_3, source_4) }
   end
 
+  context "allows filtering on association attributes" do
+    let!(:rhev)      { SourceType.create(:name => "rhev_sample",      :product_name => "RedHat Virtualization", :vendor => "redhat") }
+    let!(:openstack) { SourceType.create(:name => "openstack_sample", :product_name => "OpenStack",             :vendor => "redhat") }
+    let!(:openshift) { SourceType.create(:name => "openshift_sample", :product_name => "OpenShift",             :vendor => "redhat") }
+
+    let!(:source_rhev)       { Source.create!(:name => "rhev_source_sample",      :tenant => tenant, :source_type => rhev)      }
+    let!(:source_openstack)  { Source.create!(:name => "openstack_source_sample", :tenant => tenant, :source_type => openstack) }
+    let!(:source_openshift)  { Source.create!(:name => "openshift_source_sample", :tenant => tenant, :source_type => openshift) }
+
+    it("succeeds on a single association attribute value") do
+      expect_success("sources", "filter[source_type.name][eq]=#{rhev.name}", source_rhev)
```

Comment:
@bdunne looks like these are the old style, will need to update for the new signature, filter[source_type][name][eq]... to match the GraphQL.

Reply:
huh?

Reply:
confusing sorry, this was from the original prototype, needed to implement the filter/sort_by normalized usage instead.